src/modules/commander/commander.cpp

```diff
@@ -3050,6 +3055,7 @@ Commander::set_main_state_rc(const vehicle_status_s &status_local, const vehicle
 		 (_last_sp_man.rattitude_switch == sp_man.rattitude_switch) &&
 		 (_last_sp_man.posctl_switch == sp_man.posctl_switch) &&
 		 (_last_sp_man.loiter_switch == sp_man.loiter_switch) &&
+		 (_last_sp_man.transition_switch == sp_man.transition_switch) &&
```

Comment:
Makes me wonder if we need to categorise sticks and check for everything else being the same. That would require to stick these into two different arrays. Which we could do with a nicely crafted union.

Reply:
Yes something like that would be nice.